approval_polls/templates/approval_polls/create.html

```diff
@@ -51,17 +51,18 @@
         </div>
       </div>
       {% endfor %}
-      <div id='tag-input'>
-        <h6>(Enter a comma after each tag)</h6>
-        <input type='text' name='token-tags' class='form-control' id='tokenTagField' placeholder='Enter keywords' />
-        </label> 
-      </div> 
       <p>
         <button type='button' class='btn btn-primary btn-sm' id='add-choice'
               title='Add choice' data-toggle='tooltip' data-placement='bottom'>
           <i class='glyphicon glyphicon-plus'></i>
         </button>
       </p>
+      <div id='tag-input'>
```

Comment:
Very Minor - but could you add a `<p> </p>` around the div so that the tag box and panel borders do not touch each other. It would be nice if you could edit it in the create.html and edit.html page. Something like this.

<img width="495" alt="screen shot 2017-10-13 at 11 17 38 pm" src="https://user-images.githubusercontent.com/7693347/31573249-ddd7a904-b06c-11e7-8f74-1a3fdb255cee.png">